ADOPTERS.md

```diff
@@ -0,0 +1,21 @@
+# Paketo Buildpacks Adopters
+
+If you're using Paketo Buildpacks and want to add your organization to this list, just add a response to this Discussion and we'll get your organization added.
+
+If you want to add it yourself, please follow these directions
```

Comment:
Same. Is this supposed to link to `Adding your organization to the list of Paketo Buildpacks Adopters`? or maybe just say, "see the 'Adding your organization to the list of Paketo Buildpacks Adopters' section below`.

Reply:
Done